datastream-server/src/main/java/com/linkedin/datastream/server/Coordinator.java

```diff
@@ -1089,6 +1090,24 @@ private void hardDeleteDatastream(Datastream ds, List<Datastream> activeStreams)
     _adapter.deleteDatastream(ds.getName());
   }
 
+  /**
+   * Invokes post datastream state change action of connector for given datastream.
+   * @param datastream the datastream
+   * @throws DatastreamException if fails to perform post datastream action
+   */
+  public void invokePostDataStreamStateChangeAction(final Datastream datastream) throws DatastreamException {
+    _log.info("Invoke post datastream state change action");
+    try {
+      final String connectorName = datastream.getConnectorName();
+      final ConnectorInfo connectorInfo = _connectors.get(connectorName);
+      connectorInfo.getConnector().postDatastreamStateChangeAction(datastream);
+    } catch (DatastreamException e) {
+      _log.error("Failed to perform post datastream state change action datastream={}", datastream);
+      _metrics.updateKeyedMeter(CoordinatorMetrics.KeyedMeter.VALIDATE_DATASTREAMS_UPDATE_NUM_ERRORS, 1);
```

Comment:
What is this metric for? Is the name correct or is this a copy-paste error?

Reply:
Good catch, forgot to update with the new constant.